.github/workflows/cherry-pick-next-to-master.yml

```
@@ -0,0 +1,34 @@
+name: Cherry pick next to master
+
+on:
+  pull_request_target:
+    branches:
+      - next
+    types: ['closed']
+
+permissions: {}
+
+jobs:
+  cherry_pick_to_master:
+    runs-on: ubuntu-latest
+    name: Cherry pick into master
+    permissions:
+      pull-requests: write
+      contents: write
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'needs cherry-pick') && github.event.pull_request.merged == true }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

@mnajdova is there a reason to use the commit hash notation instead of this?

```suggestion
        uses: actions/checkout@v4.1.1
```

---

This has a great write up on the matter: https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash